pkg/meta/dapp.go

```diff
@@ -16,8 +16,8 @@ type NodePort struct {
 
 // SidecarPort represents the port for communication between node and sidecar
 type SidecarPort struct {
-	Read  int `json:"read"`
-	Write int `json:"write"`
+	LBRead  int `json:"lbRead"`
```

Comment:
Isn't this a generic structre that could be used for SC other than the load balancing?

Reply:
Nope. The broker-specifid sidecar ports won't be set by the user

Reply:
Nope. The broker-specific sidecar ports won't be set by the user